content/ja/become-a-sponsor.html

```diff
@@ -1,4 +1,49 @@
 ---
+title: スポンサーになる
 slug: become-a-sponsor
-untranslated: 1
+top_graphic: 4
+lastmod: 2019-01-28
+menu:
+main:
+weight: 40
+parent: donate
```

Comment:
I think you lost the indentation here. It must be identical to the English page between the `---` (except for the translated title of course)

Reply:
Thank you for reviewing! My IntelliJ failed to remove the indentation for .html files and I overlooked that. I'll fix it.